app/helpers/application_helper.rb

```diff
@@ -225,18 +225,29 @@ def edit_select(object, property, options = {})
 
   def flot_pie_chart(name, title, data, options = {})
     data = data.map { |k, v| {:label => k.to_s.humanize, :data => v} } if data.is_a?(Hash)
```

Comment:
Is this also the format used by the react component? or do we need to transform it as well?

Reply:
I'm :question: isn't it ment for users? This has nothing to do with users :thinking:

Reply:
sorry what? 
iirc the component takes data as `[[$label,$data]...]` instead of `[{label: $label, data: $data}...]` but i might be remembering incorrectly.

Reply:
oh, now i see that transformation on line 243, this should probably move into the condition then

Reply:
This line should move into the legacy case or it won't work for the react case